CHANGELOG.md

```diff
@@ -38,6 +38,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Improve type modeling around integer subranges.
 - Issues raised on a hard cast expression now span the entire expression in `UnicodeToAnsiCast`,
   `CharacterToCharacterPointerCast`, `NonLinearCast`, `RedundantCast`, and `PlatformDependentCast`.
+- Improve error reporting during symbol table construction.
```

Comment:
It might be good to be a little bit more specific about this, since it's
possibly something that people have come across. Maybe something like:
"Symbol table construction errors now cause indexation of the offending unit to be skipped instead of stopping the program."

(It's a little wordy, but you get the idea.)